extensions/authentication-passport/README.md

```diff
@@ -51,11 +52,53 @@ const basicStrategy = new BasicStrategy(verify);
 It's a similar configuration as you add a strategy to a `passport` by calling
 `passport.use()`.
 
-2. Apply the adapter to the strategy
+2. Supply a _user profile factory_ which converts a user to a user profile. It
+   must abide by the `UserProfileFactory` interface supplied by
+   `@loopback/authentication@3.x`.
+
+It is shown below for your convenience.
+
+```ts
+export interface UserProfileFactory<U> {
+  (user: U): UserProfile;
+}
+```
+
+A default user profile factory is provided for you in the StrategyAdapter
+constructor, but it does very little:
+
+```ts
```

Comment:
> but it does very little

Maybe add a few details like "It returns the passed in user as it is"

Reply:
+1 on rewording.